superbench/benchmarks/micro_benchmarks/cublas_function.py

```diff
@@ -8,7 +8,7 @@
 import yaml
```

Comment:
Is it separated PR for this? We can quickly merge it.

Reply:
sure, separated